repos/fdbt-site/src/pages/returnService.tsx

```diff
@@ -149,7 +149,7 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
                 const outboundDirection = directions.find((it) => ['outbound', 'clockwise'].includes(it));
                 const inboundDirection = directions.find((it) => ['inbound', 'antiClockwise'].includes(it));
 
-                if (!outboundDirection || !inboundDirection) {
+                if (inboundDirection || outboundDirection) {
```

Comment:
Why have we changed this to check if either is present? This means that we are returning any service with 1 or 2 directions - we should only be returning services with 1 direction.

Reply:
reverted to the original state